apps/web-tools/repl/components/fields/ApplicationField.tsx

```diff
@@ -1,57 +1,30 @@
 import React, { useState, useEffect } from "react";
 import Select from "react-select";
-import { getProviderId } from "@ledgerhq/live-common/manager/index";
-import ManagerAPI from "@ledgerhq/live-common/manager/api";
-import type { ApplicationVersion, DeviceInfo } from "@ledgerhq/types-live";
-import { getLatestFirmwareForDeviceUseCase } from "@ledgerhq/live-common/device/use-cases/getLatestFirmwareForDeviceUseCase";
+import type { App, DeviceInfo } from "@ledgerhq/types-live";
+import { getAppsCatalogForDevice } from "@ledgerhq/live-common/device/use-cases/getAppsCatalogForDevice";
+import { mapApplicationV2ToApp } from "@ledgerhq/live-common/apps/polyfill";
 
 export type DataTypeApplication = {
   type: "application";
 };
 
 type Props = {
-  value?: ApplicationVersion;
-  onChange: (_: ApplicationVersion | null) => void;
+  value?: App | null;
+  onChange: (_: App | null | null) => void;
```

Comment:
extran `null` here ?

Reply:
ah good catch, I need to be more careful with copilot autocompletion haha